pxr/imaging/rprUsd/CMakeLists.txt

```diff
@@ -170,9 +172,9 @@ GroupSources(rprUsd)
 
 if(RPR_ENABLE_VULKAN_INTEROP_SUPPORT)
     target_link_libraries(rprUsd ${Vulkan_LIBRARIES})
```

Comment:
Probably linkage should always be outside RPR_ENABLE_VULKAN_INTEROP_SUPPORT, please check